docs/source/extensions/logging.rst

```diff
@@ -272,21 +267,6 @@ To change this behaviour, set the ``log_every_n_steps`` :class:`~pytorch_lightni
    trainer = Trainer(log_every_n_steps=k)
 
 
-Log Writing Frequency
```

Comment:
`flush_logs_every_n_steps` is deprecated

Reply:
Instead of removing the whole section, could you add that loggers can specify their flush frequency

Reply:
I thought about doing that, but it is really an internal implementation detail of the individual loggers, and I think it makes more sense to go on the docs page for the individual loggers, since it doesn't apply to all loggers. `flush_logs_every_n_steps` is currently **only** on the CSVLogger, and this is stated on the CSVLogger doc: https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.loggers.csv_logs.html#csv-logger

Feel free to disagree though

Reply:
Hi, @ananthsub - whenever you have the time, can you take a look at the comment here? If it looks good to you, please resolve this and I can enable auto-merge if all looks good to you as well :)

Reply:
@ananthsub Can you take a look at this please so we can merge it?

Reply:
@daniellepintz as a user, I may still land on the logger section for this question. To make life easier for users, I would prefer mentioning that individual logger implementations determine their flushing behavior. This also helps users who set `flush_logs_every_n_steps` before v1.7.

Reply:
sg, updated!